ui/perf_counter_model.cpp

```diff
@@ -18,6 +18,9 @@
 #include <QDebug>
 
 #include "dive_core/available_metrics.h"
+#include "dive_core/perf_metrics_data.h"
+
+static constexpr int kFixedHeaderCount = 2;
```

Comment:
Can we keep a list of fixed header here instead of a number?
i.e. `std::array<FixedPerfMetricsDataHeaders, 2> kFixedHeader= {kDrawID, kLRZState}`